terra-base/Dockerfile

```diff
@@ -0,0 +1,178 @@
+# Smallest image with ubuntu jammy, CUDA and NVDIA drivers installed - 80 mb
+FROM --platform=linux/amd64 nvidia/cuda:12.2.0-base-ubuntu22.04
+
+#######################
+# Environment Variables
+#######################
+ENV DEBIAN_FRONTEND noninteractive
+ENV LC_ALL en_US.UTF-8
+# Ensure this matches c.NotebookApp.port in jupyter_notebook_config.py
+ENV JUPYTER_PORT 8000
+ENV JUPYTER_HOME /etc/jupyter
+ENV JUPYTER_KERNELSPEC_DIR /usr/local/share/jupyter
+# We need node >18 for jupyter to work
+ENV NODE_MAJOR 20
+
+#######################
+# Users Setup
+#######################
+# Create the jupyter user, add it to the `users` group and specify the home directory path
+ENV USER jupyter
```

Comment:
At this point, I believe your `USER` is `root`, and we are overwriting the env `USER` variable in the active environment with a different user name. This variable is actually overwritten further down this Dockerfile on this line `USER $USER` where we actively change the active user for the image.

Reply:
You are correct, this is my intention here :)